mlrun/runtimes/nuclio/application/application.py

```diff
@@ -431,6 +506,7 @@ def create_api_gateway(
         ssl_redirect: typing.Optional[bool] = None,
         set_as_default: bool = False,
         gateway_timeout: typing.Optional[int] = None,
+        port: typing.Optional[int] = None,
```

Comment:
```suggestion
        listening_port: typing.Optional[int] = None,
```

perhaps more self explanatory name for sdk

Reply:
not sure about this as it's more of a port to which we send requests when invoking. While it can be called "listening" in terms of that it's one of the listening ports of app runtime, it might confuse user a bit, so I'd stick with just `port`.